PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/util/MavenUtils.java

```diff
@@ -34,12 +34,18 @@
 import org.jetbrains.idea.maven.utils.MavenProcessCanceledException;
 
 import java.nio.file.Paths;
-import java.util.concurrent.*;
+import java.util.List;
 
 public class MavenUtils {
-    private static final ExecutorService THREAD_POOL = Executors.newCachedThreadPool();
+    public static boolean isMavenProject(Project project) {
+        return MavenProjectsManager.getInstance(project).isMavenizedProject();
+    }
+
+    public static List<MavenProject> getMavenProjects(Project project) {
+        return MavenProjectsManager.getInstance(project).getRootProjects();
+    }
 
-    public static String getSpringBootFinalJarPath(@NotNull Project ideaProject, @NotNull MavenProject mavenProject)
+    public static String getTargetFile(@NotNull Project ideaProject, @NotNull MavenProject mavenProject)
```

Comment:
It seems this method is only called by Spring Cloud Service, shall we keep it origin name or it may like a common method which could be used by Function/App Service

Reply:
I agree, this method depends on `SpringCloudDependencyManager`, I doubt whether it can also work for other type of maven projects.

Reply:
No, it is for all webapp/function and spring cloud